client/app/scripts/superdesk-archive/directives.js

```diff
@@ -396,6 +396,21 @@
                 }
             };
         }])
+        .directive('sdDraggableItem', function() {
+            return {
+                link: function(scope, elem) {
+                    if (scope.item) {
```

Comment:
It's somewhat unclear where `item` comes from, could be harder to track it in the scope hierarchy when debugging - would it make sense it to the directive explicitly so that it is immediately clear what `item` is by just looking at the HTML element the directive is used on?